clustersConfig.py

```diff
@@ -239,6 +273,16 @@ def prepare_external_port(self) -> None:
         if self.external_port == "auto":
             self.autodetect_external_port()
 
+    def validate_node_ips(self) -> bool:
+        def validate_node_ip(n: NodeConfig) -> bool:
+            if n.ip is not None:
+                if not common.ip_range_contains(self.cluster_ip_range, n.ip):
+                    logger.error(f"IP {n.ip} not in cluster subnet range for node {n.name}.")
```

Comment:
We should print the cluster subnet range to make this more explicit to users. It is not obvious to me that if I do not specify a cluster_ip range, I will only be able to specify nodeIPs within the first n IPs where n=number of nodes.

Reply:
+1, the logging needs work. I want to read 1 sentence and get an as-good-as-possible understanding of what's up.

Reply:
Should be fixed now, good point!